app/controllers/posts_controller.rb

```diff
@@ -23,12 +23,14 @@ def create
   def index
     @posts = current_user.posts.order(created_at: :desc)
     @user_likes = current_user.likes.where(post_id: @posts.map(&:id)).index_by(&:post_id)
+    @users = User.all
+    @follow = User.where(user_id: current_user.id)
   end
 
   def edit
-    unless current_user == @post.user
-      redirect_to root_path, alert: 'You are not authorized to edit this post.'
-    end
+    return if current_user == @post.user
+
+    redirect_to root_path, alert: 'You are not authorized to edit this post.'
   end
```

Comment:
Why do you remove the past code? What is the purpose of making a return?

Reply:
jajja ahhh Idk why that is again, maybe on a PR I guess let me solve it